src/components/toastrs/toastr/toastr.js

```diff
@@ -79,7 +79,7 @@ class Toastr extends React.Component {
         message = (
           <Message
             success
-            styleName='toastr'
+            styleName='success-toastr'
             icon='checkmark box'
             header={header}
             content={content}
```

Comment:
cant these be a function in the toastr.js file instead? seems like to much separation